common/djangoapps/contentserver/caching.py

```diff
@@ -0,0 +1,47 @@
+"""
+Helper functions for caching course assets.
+"""
+from django.core.cache import get_cache
+from opaque_keys import InvalidKeyError
+
+CONTENTSERVER_VERSION=1
+
+# See if there's a "course_assets" cache configured, and if not, fallback to the default cache.
+try:
+    cache = get_cache('course_assets')
```

Comment:
`get_cache()` has been deprecated since Django 1.7. Please use [`caches`](https://docs.djangoproject.com/en/1.8/topics/cache/#django.core.cache.caches)